Services/AccessControl/interfaces/interface.ilAccessHandler.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * Interface ilAccessHandler
+ *
+ * This interface combines all available interfaces whic can be called via global $ilAccess
+ *
+ * @author Fabian Schmid <fs@studer-raimann.ch>
+ */
+interface ilAccessHandler extends ilRBACAccessHandler, ilOrgUnitPositionAccessHandler, ilOrgUnitPositionAndRBACAccessHandler {
```

Comment:
@smeyer-ilias as told you during our phone call i renamed ilAccessHandler to ilAccess and introduced the combining interface ilAccessHandler